tests/compat.sh

```diff
@@ -1408,12 +1408,12 @@ for VERIFY in $VERIFIES; do
                     reset_ciphersuites
                     if [ `minor_ver "$MODE"` -ge 4 ]
                     then
-                        M_CIPHERS="$M_CIPHERS               \
-                            TLS_AES_128_GCM_SHA256          \
-                            TLS_AES_256_GCM_SHA384          \
-                            TLS_AES_128_CCM_SHA256          \
-                            TLS_AES_128_CCM_8_SHA256        \
-                            TLS_CHACHA20_POLY1305_SHA256    \
+                        M_CIPHERS="$M_CIPHERS                  \
+                            TLS1-3-AES-128_GCM-SHA256          \
+                            TLS1-3-AES-256_GCM-SHA384          \
+                            TLS1-3-AES-128_CCM-SHA256          \
+                            TLS1-3-AES-128_CCM-8-SHA256        \
+                            TLS1-3-CHACHA20_POLY1305-SHA256    \
```

Comment:
Need change underscore (`_`) to dash(`-`) to match the name in `ssl_ciphersuites.c`
                            TLS1-3-AES-128-GCM-SHA256          \
                            TLS1-3-AES-256-GCM-SHA384          \
                            TLS1-3-AES-128-CCM-SHA256          \
                            TLS1-3-AES-128-CCM-8-SHA256        \
                            TLS1-3-CHACHA20-POLY1305-SHA256    \

Reply:
Yes :+1:

Reply:
LGTM.